packages/ast-spec/tsconfig.build.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
     "composite": true,
+    "jsx": "preserve",
```

Comment:
Otherwise:

```plaintext
> nx run ast-spec:typecheck

$ tsc -p tsconfig.json --noEmit
src/jsx/JSXNamespacedName/fixtures/component/fixture.tsx:1:11 - error TS17004: Cannot use JSX unless the '--jsx' flag is provided.

1 const x = <NamespacePropComponent a:b="tight spacing" />;
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

https://github.com/typescript-eslint/typescript-eslint/actions/runs/5194185770/jobs/9365660553?pr=7088